src/content/changes/highlights/2022-09-15.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2022-09-15T10:00:00
+title: Highlights for the week ending September 15, 2022
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+
+
+## User interfaces
+
+
+
+## Apps
```

Comment:
```suggestion
## User interfaces

In kubectl-gs we have deprecated some flags in three commands, to avoid name conflicts with global flags (which are usable in all commands). You will see a warning when using the deprecated flags. This affects the following commands:

- `kubectl gs template app`: flag `--target-namespace` replaces `--namespace`, `--cluster-name` replaces `--cluster`.
- `template catalog`: flag `--target-namespace` replaces `--namespace`.
- `gitops add app`: flag `--target-namespace` replaces `--namespace`.

Please move any scripts to the new flags, as the old flags will change their meaning with the next major release of kubectl-gs.

kubectl-gs now also provides a command `get organizations`, which is a way for all management API users to list the organizations they have access to. `kubectl get organizations` in contrast requires admin permissions.

## Apps
```